### ISSUE TITLE
Versioning is not saving

### ISSUE THREAD

**Issue by BSWG** [driftyco/ionic-plugin-deploy#125]
If I edit the versioning of an active deploy and press Save and reopen the dialog the versioning information is lost (max version). All versions of the app get the last active deploy, which is not preferable for the latest binary release.

**Comment by piotrowski** [comment]
This is feedback for the app.ionic.io Deploy interface, right?

**Comment by BSWG** [comment]
Yes. That is correct. But I couldn't find the place for that kind of feedback. I hoped the deploy developers for cli and the cloud service would be on the same team?

**Comment by piotrowski** [comment]
They might - but I think a better location for that report might be here: https://ionic.zendesk.com/hc/en-us/requests/new?ticket_form_id=632048 (via http://ionicframework.com/support)
